pkg/virtctl/vm/add_volume_test.go

```diff
@@ -185,3 +161,29 @@ var _ = Describe("Add volume command", func() {
 		Entry("addvolume dv, with LUN-type disk should call VMI endpoint", "addvolume", "testvmi", "testvolume", true, expectVMIEndpointAddVolume, "--disk-type", "lun"),
 	)
 })
+
+func createTestDataVolume() *v1beta1.DataVolume {
+	return &v1beta1.DataVolume{
+		ObjectMeta: k8smetav1.ObjectMeta{
+			Name: "testvolume",
+		},
+	}
+}
+
+func createTestPVC() *k8sv1.PersistentVolumeClaim {
+	return &k8sv1.PersistentVolumeClaim{
+		ObjectMeta: k8smetav1.ObjectMeta{
+			Name: "testvolume",
+		},
+	}
+}
+
+func verifyVolumeSource(volumeOptions *v1.AddVolumeOptions, useDv bool) {
+	if useDv {
```

Comment:
Add `Expect(volumeOptions).ToNot(BeNil())` and `Expect(volumeOptions.VolumeSource).ToNot(BeNil())`?

Reply:
I prefer to not do it in this PR, this PR just converted to not inline
if it is nil it will panic on the other fields, when it deref